transcendence-app/src/shared/local-file/infrastructure/db/postgres/local-file.postgres.repository.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@nestjs/common';
+
+import { BasePostgresRepository } from '../../../../../shared/db/postgres/postgres.repository';
+import { table } from '../../../../../shared/db/models';
+import { LocalFileEntity, LocalFileKeys } from '../local-file.entity';
+import { PostgresPool } from '../../../../../shared/db/postgres/postgresConnection.provider';
```

Comment:
'../../../../../' <- this is still horrible, but it's also out of the scope of this PR.